backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Graphics.java

```diff
@@ -534,6 +534,16 @@ public void setSystemCursor (SystemCursor systemCursor) {
 		Lwjgl3Cursor.setSystemCursor(getWindow().getWindowHandle(), systemCursor);
 	}
 
+	public void setIdleFPS (int idleFPS) {
+		getWindow().getConfig().idleFPS = idleFPS;
```

Comment:
Are you sure that does anything? Lwjgl3Application seems to use the inital configuration and not the one provided by the windows: https://github.com/libgdx/libgdx/blob/916663ed3d4ebd71c3bc83e6c8d3483c3aa37612/backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Application.java#L241